Packs/CortexAttackSurfaceManagement/README.md

```diff
@@ -15,6 +15,17 @@ This pack contains all of the integrations, automations, and playbooks necessary
 - Supports validation re-scanning to ensure that remediation efforts have been applied successfully.
 - Includes PDF reporting capabilities for preserving and communicating the investigation summary.
 
+## How to use it?
+
+The Active Response playbook contains a set of sub-playbooks, which support many different remediation paths that can be taken depending on the types of configured integrations, the type of alert, and input provided by the analyst.
+
+For setting up the Active Response module for Xpanse, [a guide on how to configure the Active Response module can we found here.](https://docs-cortex.paloaltonetworks.com/r/Cortex-XPANSE/Cortex-Xpanse-Expander-User-Guide/Set-Up-Active-Response)
+
+Aditionally, [a list of integrations used for the Active Response playbook can be found here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XPANSE/Cortex-Xpanse-Expander-User-Guide/Automated-Remediation-Capabilities-Matrix?section=UUID-0a5dcbc2-d5ab-fa4e-5efc-599daac8b39b_table-idm4546555537995233526554598204). That are needed for different enrichment and remediation possibilities.
```

Comment:
May be use "These are needed for different enrichment and remediation possibilities." instead of "That are needed for different enrichment and remediation possibilities."